src/include/rollback_to_stable.h

```diff
@@ -6,6 +6,36 @@
  * See the file LICENSE for redistribution information.
  */
 
+/*
+ * Helper macros for finer-grained RTS verbose messaging categories.
+ */
+#define WT_RTS_VERB_TAG_FILE_SKIP "[FILE_SKIP] "
+#define WT_RTS_VERB_TAG_HS_GT_ONDISK "[HS_GT_ONDISK] "
+#define WT_RTS_VERB_TAG_HS_STOP_OBSOLETE "[HS_STOP_OBSOLETE] "
+#define WT_RTS_VERB_TAG_HS_ABORT_STOP "[HS_ABORT_STOP] "
+#define WT_RTS_VERB_TAG_HS_RESTORE_TOMBSTONE "[HS_RESTORE_TOMBSTONE] "
```

Comment:
This list isn't correctly sorted by alphabet in the HS section - could you fix it up?